msa/tb/docker-postgres/start-db.sh

```diff
@@ -22,14 +22,22 @@ PG_CTL=$(find /usr/lib/postgresql/ -name pg_ctl)
 if [ ! -d ${PGDATA} ]; then
     mkdir -p ${PGDATA}
     ${PG_CTL} initdb
+else
+    ${PG_CTL} start
```

Comment:
I'm not sure why we need to do `${PG_CTL} start`, because the Postgres service starts automatically with no problems. Maybe there is another reason, so please, share your experience regarding that line.